src/Assert.php

```diff
@@ -817,6 +818,24 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $class, $message = '')
+    {
+        static::string($class);
+
+        try {
+            $expression();
+        } catch (Exception $e) {
```

Comment:
indeed, as commented, we should catch `Throwable` as well.

```php
try {

} catch (\Exception $e) {

} catch (\Throwable $e) {

}
```

will work on both PHP 5 and PHP 7